src/common/components/paymentMethods/paymentMethodDisplay.tpl.html

```diff
@@ -28,8 +28,11 @@
         ng-switch-when="Diners Club"
       />
     </span>
-    <strong>{{$ctrl.paymentMethod['card-type']}}</strong>
-    <span>ends in ****{{$ctrl.paymentMethod['last-four-digits']}}</span>
+    <strong
```

Comment:
This change makes the entire line bold instead of just the card type. I think the `id``needs to go on a new wrapper element.